<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gerenciador</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="filter_text">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>491</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_filter">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Filtrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pid_text_all">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_kill">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>KILL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_stop">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_cont">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>350</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>CONT</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>310</y>
      <width>241</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Advancend</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>241</width>
       <height>121</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Priority</string>
      </attribute>
      <widget class="QPushButton" name="b_priority">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pid_text_priority">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PRIORITY</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="priority_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>CPU SET</string>
      </attribute>
      <widget class="QPushButton" name="b_cpuset">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>31</width>
         <height>25</height>
        </rect>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="pid_text_cpu">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>10</y>
         <width>161</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>PID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>67</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>NUMBER</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="cpu_text">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>81</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QTableWidget" name="list_process">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1051</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
